src/Pagination/Pagination.test.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import { render, fireEvent } from '@testing-library/react'
+
+import Pagination from './index'
+import { get } from 'http'
```

Comment:
wut

Reply:
Idk what that is or how that got there. But I'm deeply saddened by it.